resources/sass/components/forum/_post.scss

```diff
@@ -143,6 +143,16 @@
   table {
     @extend .table-responsive-sm;
   }
+
+  blockquote {
```

Comment:
czy to dotyczy jedynie forum czy rowniez pozostalych czesci serwisu?

w linii 225 tegoz pliku jest juz kod dla `<blockquote>`.
Tak poza tym, ten kod znajduje się w pliku `_posts.scss` co sugeruje że odnosi się jedynie do postów na forum. Jest to jednak znacznik globalny i ten styl będzie aplikowany do wszystkich cytatów w serwisie. Może przenieść do nowego pliku - np. `_blockquote.scss`?

Reply:
@adam-boduch Plan był żeby tylko cytaty w postach na forum były zmienione.

Reply:
Style są w `.post-content`, więc myślałem że odnosi się tylko do treści postu.